internal/datastore/postgres/common/pgx.go

```diff
@@ -102,7 +103,7 @@ func ConfigurePGXLogger(connConfig *pgx.ConnConfig) {
 		}
 	}
 	l := zerologadapter.NewLogger(log.Logger)
-	connConfig.Logger = levelMappingFn(l)
+	connConfig.Tracer = &tracelog.TraceLog{Logger: levelMappingFn(l), LogLevel: tracelog.LogLevelInfo}
```

Comment:
hmm, what does `LogLevel` mean here? should this be aligned with the log level of the application instead of hardcoded to info?

Reply:
the `LogLevel` here just sets whether we should log all messages (info) or errors only (error). 

we already do the mapping from info -> debug here so I think it makes sense to leave it